spring-cloud-dataflow-docs/src/main/asciidoc/applications.adoc

```diff
@@ -0,0 +1,8 @@
+[[applications]]
+= Applications
+
+[partintro]
+--
+A selection of pre-built link:http://cloud.spring.io/spring-cloud-stream-app-starters/[stream] and link:http://cloud.spring.io/spring-cloud-task-app-starters/[task/batch] starter apps for various data integration and processing scenarios facilitate learning and experimentation. For more details, review how to <<streams.adoc#spring-cloud-dataflow-register-stream-apps, register applications>>
```

Comment:
The link to the app starters page doesn't provide enough SCDF context.  One has to scroll down to see the section `Stream App Starters and Spring Cloud Data Flow` which has the bit.ly links and then there is another link with the `**` that points *back* to SCDF docs on how to register them.

This section should provide a description of the bit.ly links (show some sample contents) and show how to register one of the bit.ly links.  Also, we can link to the app-registration section, which I notice does not include registering metadata URLs.